control-plane/cni/main_test.go

```diff
@@ -355,3 +377,65 @@ const missingIPsStdinData = `{
     "name": "consul-cni",
     "type": "consul-cni"
 }`
+
+// TODO: need to make this match what Nomad emits
```

Comment:
Is this resolved since you mentioned you tested it end to end?

Reply:
:facepalm: Yes, thanks. Will remove!